test/unit/helpers/logger/external-logger-bigquery-test.js

```diff
@@ -42,10 +43,12 @@ describe("External Logger", () => {
       display_id: displayId,
       cache_version: version,
       os: os,
-      ts: date.toISOString()
+      ts: date.toISOString(),
+      file_url: "",
```

Comment:
If default parameters are used, then it wouldn't be necessary to set these initial values.